docs/docs/getting-started.md

```diff
@@ -27,7 +27,11 @@ The Hello Qlik Core examples are divided into three separate parts:
     [enigma.js](https://github.com/qlik-oss/enigma.js/) to communicate with it.
 - [Hello Data](./tutorials/hello-data.md) - Loading user data into Qlik Associative Engine using
     [halyard.js](https://github.com/qlik-oss/halyard.js).
-- [Hello Visualization](./tutorials/hello-visualization.md) - Building a visualization using picasso.js.
+- [Hello Visualization](./tutorials/hello-visualization.md) - Building visualizations using [picasso.js](https://picassojs.com/).
+
+### Screenshot
```

Comment:
Instead of a new headers, perhaps just a new paragraph explaining the screenshot as the end result of the hello tutorials? Like `Once you have completed these tutorials, you should be presented with a result similar to the screenshot below.`